package.json

```diff
@@ -7,10 +7,12 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "test-ci": "ng test --no-watch --no-progress --code-coverage",
+    "test-ci": "ng test --no-watch --no-progress --code-coverage --browsers=ChromeHeadless",
```

Comment:
We can also think on using this as a standard for all tests (ci and local)

Reply:
We probably need a karma config anyway and can put that there then. I'd probably do that in a separate PR.